website/content/docs/configuration/listener/tcp.mdx

```diff
@@ -369,7 +401,137 @@ api_addr = "https://[2001:1c04:90d:1c00:a00:27ff:fefa:58ec]:8200"
 cluster_addr = "https://[2001:1c04:90d:1c00:a00:27ff:fefa:58ec]:8201"
 ```
 
+## Redaction examples
+
+Please see redaction settings above, for details on each redaction setting.
+
+Example configuration for the [`tcp`](/vault/docs/configuration/listener/tcp) listener,
+enabling [`redact_addresses`](/vault/docs/configuration/listener/tcp#redact_addresses),
+[`redact_cluster_name`](/vault/docs/configuration/listener/tcp#redact_cluster_name) and
+[`redact_version`](/vault/docs/configuration/listener/tcp#redact_version).
+
+```hcl
+ui            = true
+cluster_addr  = "https://127.0.0.1:8201"
+api_addr      = "https://127.0.0.1:8200"
+disable_mlock = true
+
+storage "raft" {
+  path = "/path/to/raft/data"
+  node_id = "raft_node_1"
+}
+
+listener "tcp" {
+  address             = "127.0.0.1:8200",
+  tls_cert_file = "/path/to/full-chain.pem"
+  tls_key_file  = "/path/to/private-key.pem"
+  redact_addresses    = "true"
+  redact_cluster_name = "true"
+  redact_version      = "true"
+}
+
+telemetry {
+  statsite_address = "127.0.0.1:8125"
+  disable_hostname = true
+}
+```
+
+### API: `/sys/health`
+
+`curl -s https://127.0.0.1:8200/v1/sys/health | jq`:
+
+```json
+{
+  "initialized": true,
+  "sealed": false,
+  "standby": true,
+  "performance_standby": false,
+  "replication_performance_mode": "disabled",
+  "replication_dr_mode": "disabled",
+  "server_time_utc": 1696598650,
+  "version": "",
+  "cluster_id": "a1a7a078-0ae1-7fb9-41ec-2f4f583c773e"
+}
+```
+
+When comparing the JSON response above to when redaction settings are not enabled,
+we can see that `cluster_name` is not present (it is now omitted due to being empty).
+Also, that `version` is now redacted and shown only as `""`.
+
+### API: `sys/leader`
+
+`curl -s https://127.0.0.1:8200/v1/sys/leader | jq`:
+
+```json
+{
+  "ha_enabled": true,
+  "is_self": false,
+  "active_time": "0001-01-01T00:00:00Z",
+  "leader_address": "",
+  "leader_cluster_address": "",
+  "performance_standby": false,
+  "performance_standby_last_remote_wal": 0,
+  "raft_committed_index": 164,
+  "raft_applied_index": 164
+}
+```
+
+When comparing the JSON response above to when redaction settings are not enabled,
+we can see that `leader_address` and `leader_cluster_address` are now redacted and
+shown only as `""`.
+
+### API: `sys/seal-status`
+
+`curl -s https://127.0.0.1:8200/v1/sys/seal-status | jq`:
+
+```json
+{
+  "type": "shamir",
+  "initialized": true,
+  "sealed": false,
+  "t": 1,
+  "n": 1,
+  "progress": 0,
+  "nonce": "",
+  "version": "",
+  "build_date": "",
+  "migration": false,
+  "cluster_id": "a1a7a078-0ae1-7fb9-41ec-2f4f583c773e",
+  "recovery_seal": false,
+  "storage_type": "raft"
+}
+```
+
+When comparing the JSON response above to when redaction settings are not enabled,
+we can see that `cluster_name` is not present (it is now omitted due to being empty).
+Also, that `build_date` and `version` are now redacted and shown only as `""`.
+
+### CLI: `vault status`
+
```

Comment:
```suggestion
The CLI command `vault status` uses endpoints that support redacting data, so the output redacts `Version`, `Build Date`, `HA Cluster`, and `Active Node Address` .`Version`, `Build Date`, `HA Cluster` show `n/a` because the underlying endpoint returned the empty string, and  `Active Node Address` shows as `<none>` because it was omitted from the API response.
```